### PR TITLE
new commit

### PR DESCRIPTION
**Pull request by riabtsun**

### CODE REVIEW

File: icon_font/_icon-font.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
+@font-face {
+font-family: 'icomoon';
+src: url('/icon_font/icomoon.woff2') format('woff2');
+src: url('/icon_font/icomoon.woff') format('woff');
+font-weight: normal;
+font-style: normal;
+font-display: block;
```
Comment (Plakhotenko) [inline icon_font/_icon-font.scss]: Please, add indents.
Check all the files.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    rel="stylesheet">
+  <title>Best Tour Plan</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="service-header">
+      <div class="container">
+        <div class="row top-header">
+          <div class="head-info col-12 col-lg-6">
```
Comment (Plakhotenko) [inline index.html]: You don't need col-12. Check all the document.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+            <a href="#" class="head-info__item">
+              <span class="icon icon-present gift_icon"></span>
+              <span>BTP Gift Card</span>
+            </a>
+          </div>
+          <div class="col-12 col-sm-6 col-lg-3 top-header__link">
+            <a href="#" class="head-info__item">
+              <span class="icon icon-star black_star"></span>
+              <span>Loyalty Program</span>
+            </a>
```
Comment (Plakhotenko) [inline index.html]: Please, check class naming according to BEM.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+        <div class="row mid-header">
+          <div class="header-logo col-12 col-md-3">
+            <a href="#" aria-label="main page link" class="logo_link"><img src="./images/header_logo.png"
+                alt="logo"></a>
+          </div>
+          <div class="col-12 col-md-4 offset-md-1">
+            <form class="search-form">
+              <input class="search-form__text" type="search" name="search" placeholder="Search Location">
+              <button class="search-form__btn" type="submit" value="search">
+                <span class="icon icon-search search-form__btn--icon"></span>
```
Comment (Plakhotenko) [inline index.html]: Maybe search-form__btn-icon

File: index.html [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+              <span class="icon icon-Stroke dropdown__stroke "></span>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <nav class="nav">
+      <div class="container">
+        <div class="row nav-header">
+          <div class="bot-header">
```
Comment (Plakhotenko) [inline index.html]: Why do you need so many wrappers?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+          </div>
+        </div>
+      </div>
+    </nav>
+    </div>
+    </div>
+  </header>
+
+  <main>
+    <section class="section top-section">
```
Comment (Plakhotenko) [inline index.html]: Maybe only top-section?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+      <div class="container">
+        <nav aria-label="breadcrumb">
+          <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="#">Home</a></li>
+            <li class="breadcrumb-item"><a href="#">Flash Offers</a></li>
+            <li class="breadcrumb-item" aria-current="page">Grand Hilton Hotel</li>
+          </ol>
+        </nav>
```
Comment (Plakhotenko) [inline index.html]: We should use `nav` tag only for main navigation.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+        <div class="row">
+          <div class="col">
+            <div class="hotel-descr">
+              <div class="hotel__stars">
+                <span class="icon icon-gold-star gold-star"></span>
+                <span class="icon icon-gold-star gold-star"></span>
+                <span class="icon icon-gold-star gold-star"></span>
+                <span class="icon icon-gold-star gold-star"></span>
+                <span class="icon icon-gold-star gold-star"></span>
```
Comment (Plakhotenko) [inline index.html:130]: Maybe just "star" and "star-alt".

File: index.html [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+              </div>
+            </div>
+            <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+              <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+              <span class="sr-only">Previous</span>
+            </a>
+            <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+              <span class="carousel-control-next-icon" aria-hidden="true"></span>
+              <span class="sr-only">Next</span>
+            </a>
```
Comment (Plakhotenko) [inline index.html]: need refactoring

File: index.html [hunk 10]
```diff
@@ -0,0 +1,175 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="choose online tour for trip">
+  <link rel="stylesheet" href="./scss/bootstrap.scss">
```
Comment (Plakhotenko) [inline index.html:8]: Only CSS

File: index.html [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+      </div>
+    </section>
+  </main>
+
+  <!-- Optional JavaScript -->
+  <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+  <script src="./jquery/jquery-3.5.1.slim.min.js"></script>
+  <script src="./bootstrap/js/dist/index.js"></script>
+  <script src="./bootstrap/js/dist/util.js"></script>
+  <!-- <script src="./bootstrap/js/js/bootstrap.min.js"></script> -->
```
Comment (Plakhotenko) [inline index.html]: Please, remove unnecessary comments.

File: styles/_body.scss [hunk 12]
```diff
@@ -0,0 +1,13 @@
+body {
+  font-family: 'Mulish', sans-serif;
+}
+a {
+  color: #333;
```
Comment (Plakhotenko) [inline styles/_body.scss:5]: Are you sure?

File: styles/_header.scss [hunk 13]
```diff
@@ -0,0 +1,188 @@
...
+  background: #f5fafe;
+  font-weight: 400;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  justify-content: space-between;
+  text-align: center;
+  @include media-breakpoint-up (lg) {
+    text-align: justify;
+  }
+}
```
Comment (Plakhotenko) [inline styles/_header.scss]: Did you miss `display: flex` ?

File: styles/_header.scss [hunk 14]
```diff
@@ -0,0 +1,188 @@
...
+    line-height: 13px;
+  }
+}
+.head-info__item {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 10px;
+  @include media-breakpoint-up(md)
+  {
```
Comment (Plakhotenko) [inline styles/_header.scss]: Check formatting.

File: styles/_header.scss [hunk 15]
```diff
@@ -0,0 +1,188 @@
...
+  display: none;
+  position: relative;
+  font-weight: 600;
+  font-size: 22px;
+  @include media-breakpoint-up(md) {
+    display: flex;
+  }
+}
+.shop-cart::after {
+  content: '2';
```
Comment (Plakhotenko) [inline styles/_header.scss]: Move to html.

**Review by Plakhotenko — CHANGES_REQUESTED** [review]

**Comment by Plakhotenko** [comment]
@riabtsun Please check title and description of the pull request.

File: icon_font/_icon-font.scss (hunk 1 above)
Comment (riabtsun) [reply icon_font/_icon-font.scss:7]: indents added

File: styles/_header.scss (hunk 15 above)
Comment (riabtsun) [reply styles/_header.scss]: > Move to html.
did it

File: styles/_header.scss (hunk 14 above)
Comment (riabtsun) [reply styles/_header.scss]: > Check formatting.

corrected

File: styles/_header.scss (hunk 13 above)
Comment (riabtsun) [reply styles/_header.scss]: > Did you miss `display: flex` ?

i found it )

File: styles/_body.scss (hunk 12 above)
Comment (riabtsun) [reply styles/_body.scss:5]: > Are you sure?

it will be change in future

File: index.html (hunk 11 above)
Comment (riabtsun) [reply index.html]: > Please, remove unnecessary comments.

did it

File: index.html (hunk 9 above)
Comment (riabtsun) [reply index.html]: > need refactoring

ok

File: index.html (hunk 8 above)
Comment (riabtsun) [reply index.html:130]: > Maybe just "star" and "star-alt".

ok, next time

File: index.html (hunk 7 above)
Comment (riabtsun) [reply index.html]: > We should use `nav` tag only for main navigation.

corrected

File: index.html (hunk 6 above)
Comment (riabtsun) [reply index.html]: > Maybe only top-section?

of course, corrected

File: index.html (hunk 5 above)
Comment (riabtsun) [reply index.html]: > Why do you need so many wrappers?

excess wrap removed

File: index.html (hunk 4 above)
Comment (riabtsun) [reply index.html]: > Maybe search-form__btn-icon

renamed

File: index.html (hunk 3 above)
Comment (riabtsun) [reply index.html]: > Please, check class naming according to BEM.

classes renamed

File: index.html (hunk 2 above)
Comment (riabtsun) [reply index.html]: > You don't need col-12. Check all the document.

col-12 classes removed

File: index.html (hunk 10 above)
Comment (riabtsun) [reply index.html:8]: > Only CSS

scss link removed

**Review by riabtsun — COMMENTED** [review]
fixes some bugs

File: icon_font/_icon-font.scss (hunk 1 above)
Comment (riabtsun) [reply icon_font/_icon-font.scss]: > Please, add indents.
> Check all the files.
indents added

File: icon_font/_icon-font.scss [hunk 16]
```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: 'icomoon';
+  src: url('/icon_font/icomoon.woff2') format('woff2');
+  src: url('/icon_font/icomoon.woff') format('woff');
```
Comment (Plakhotenko) [inline icon_font/_icon-font.scss:4]: Use relative path.

Comment (Plakhotenko) [inline icon_font/_icon-font.scss:4]: Do we really need both of woff2 and woff?

File: styles/_header.scss [hunk 17]
```diff
@@ -0,0 +1,192 @@
...
+}
+.mid-header {
+  display: flex;
+  align-items: center;
+  padding-top: 40px;
+  padding-bottom: 40px;
+
+}
+.header-logo {
+text-align: center;
```
Comment (Plakhotenko) [inline styles/_header.scss:57]: fix indent

**Comment by Plakhotenko** [comment]
Please, create another PR for new section.
And this PR should be merged.

File: styles/_top_section.scss [hunk 18]
```diff
@@ -0,0 +1,216 @@
...
+  line-height: 10px;
+  color: #fff;
+}
+.raiting-info__grade {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 18px;
+  color: #fff;
+}
+#carouselExampleControls {
```
Comment (Plakhotenko) [inline styles/_top_section.scss:87]: Don't use id in css.

File: styles/_top_section.scss [hunk 19]
```diff
@@ -0,0 +1,216 @@
...
+  display: flex;
+  flex-wrap: wrap;
+}
+.booking-info {
+  display: flex;
+  flex-direction: column;
+  max-height: 210px;
+}
+.booking-info__item-1 {
+  font-family: 'Nunito', sans-serif;
```
Comment (Plakhotenko) [inline styles/_top_section.scss:103]: Use variables

File: styles/_top_section.scss [hunk 20]
```diff
@@ -0,0 +1,216 @@
...
+.room-info__item-3 {
+  font-family: 'Nunito', sans-serif;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 25px;
+  color: #fff;
+}
+.guest-icon,
+.room-icon,
+.icon-phone {
```
Comment (Plakhotenko) [inline styles/_top_section.scss:147]: Can we use common class?

File: styles/_top_section.scss [hunk 21]
```diff
@@ -0,0 +1,216 @@
...
+  }
+}
+.room-info {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: flex-end;
+  align-content: center;
+}
+.booking-card__btn {
```
Comment (Plakhotenko) [inline styles/_top_section.scss:174]: There is the booking-card block? Is it exist?

File: icon_font/_icon-font.scss [hunk 22]
```diff
@@ -0,0 +1,79 @@
...
+  font-family: 'icomoon' !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+-webkit-font-smoothing: antialiased;
+-moz-osx-font-smoothing: grayscale;
```
Comment (Plakhotenko) [inline icon_font/_icon-font.scss:20]: fix indent

File: index.html [hunk 23]
```diff
@@ -0,0 +1,409 @@
...
+                <li>
+                  <span class="icon icon-chekin check"></span>
+                  <span class="hightlight-list__text">Glass bottom boat trips</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="row details-box-2">
+            <div class="col">
+              <ul class="details-box__list">
```
Comment (Plakhotenko) [inline index.html:358]: There is details-box block?

File: styles/_header.scss [hunk 24]
```diff
@@ -0,0 +1,195 @@
...
+  @include media-breakpoint-up(lg) {
+  font-size: 10px;
+  line-height: 13px;
+  }
+}
+.head-info__phone {
+  font-weight: 700;
+}
+.gift_icon,
+.black_star {
```
Comment (Plakhotenko) [inline styles/_header.scss:17]: Can we use common class?

File: styles/_package_section.scss [hunk 25]
```diff
@@ -0,0 +1,104 @@
+.package-section {
+  background: #f5fafe;
+}
+.package__h2 {
```
Comment (Plakhotenko) [inline styles/_package_section.scss:4]: Maybe package__title. Don't use this naming way.

File: styles/_package_section.scss [hunk 26]
```diff
@@ -0,0 +1,104 @@
...
+  margin-bottom: 0;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  margin-bottom: 20px;
+  @include media-breakpoint-up(md){  
+    column-count: 2;
+  }
+}
+.offer-list {
+margin-bottom: 0;
```
Comment (Plakhotenko) [inline styles/_package_section.scss:40]: indent

File: styles/_package_section.scss [hunk 27]
```diff
@@ -0,0 +1,104 @@
...
+  @include media-breakpoint-up(xl) {
+    padding-top: 48px;
+  }    
+}
+.hightlight-list__item {
+  padding-bottom: 24px;
+}
+.hightlight-list__item:nth-child(3) {
+  @include media-breakpoint-up(lg){
+padding-bottom: 0;
```
Comment (Plakhotenko) [inline styles/_package_section.scss:83]: indent

File: styles/_package_section.scss [hunk 28]
```diff
@@ -0,0 +1,104 @@
...
+  }
+}
+.details-box {
+  background-color: #fff;
+  border-bottom: 1px dashed #E0E0E0; 
+}
+.details-box-2 {
```
Comment (Plakhotenko) [inline styles/_package_section.scss:90]: I guess it should be modifier. Like details-box--2

File: styles/_top_section.scss [hunk 29]
```diff
@@ -0,0 +1,216 @@
...
+.hotel-descr {
+  padding-right: 0;
+  padding-left: 0;
+}
+.breadcrumb {
+  background: #f5fafe;
+  padding: 0;
+  margin-bottom: 42px;
+}
+.hotel__stars {
```
Comment (Plakhotenko) [inline styles/_top_section.scss:20]: There is the hotel block?

**Review by Plakhotenko — CHANGES_REQUESTED** [review]

**Pull request merged**